python/paddle/distributed/fleet/data_generator/data_generator.py

```diff
@@ -156,13 +156,13 @@ def generate_sample(self, line):
 
         Returns:
```

Comment:
Returns:
            Returns the data processed by the user.
            The data format is list or tuple:[(name, [feasign, ...]), ...]
            or ((name, [feasign, ...]), ...)